src/storage/txn/process.rs

```diff
@@ -768,6 +768,26 @@ fn process_write_impl<S: Snapshot>(
 
             (pr, modifies, rows, ctx, None)
         }
+        Command::ResolveLockLite {
+            ctx,
+            start_ts,
+            commit_ts,
+            resolve_keys,
+        } => {
+            let mut txn = MvccTxn::new(snapshot.clone(), start_ts, !ctx.get_not_fill_cache())?;
+            let rows = resolve_keys.len();
+            // ti-client guarantees the size of resolve_keys will not too large, so no necessary
+            // to control the write_size as ResolveLock.
+            for key in resolve_keys {
+                if commit_ts > 0 {
+                    txn.commit(key, commit_ts)?;
+                } else {
+                    txn.rollback(key)?;
```

Comment:
```suggestion
                    is_pessimistix_txn = txn.rollback(key)?;
```

Reply:
Is it possible some keys return true and some keys return false?

Reply:
done